hack/templates/values/values.template

```diff
@@ -228,53 +280,76 @@ ofedDriver:
   #       cpu: "300m"
   #       memory: "300Mi"
   terminationGracePeriodSeconds: 300
-  # Private mirror repository configuration
+  # -- Private mirror repository configuration.
   repoConfig:
     name: ""
-  # Custom ssl key/certificate configuration
+  # Custom ssl key/certificate configuration.
   certConfig:
+    # -- Custom TLS key/certificate configuration configMap name.
     name: ""
   startupProbe:
+    # -- NVIDIA DOCA Driver startup probe initial delay.
     initialDelaySeconds: 10
+    # -- NVIDIA DOCA Driver startup probe interval.
     periodSeconds: 20
   livenessProbe:
+    # -- NVIDIA DOCA Driver liveness probe initial delay.
     initialDelaySeconds: 30
+    # -- NVIDIA DOCA Driver liveness probe interval.
     periodSeconds: 30
   readinessProbe:
+    # -- NVIDIA DOCA Driver readiness probe initial delay.
     initialDelaySeconds: 10
+    # -- NVIDIA DOCA Driver readiness probe interval.
     periodSeconds: 30
   upgradePolicy:
-    # global switch for automatic upgrade feature
-    # if set to false all other options are ignored
+    # -- Global switch for automatic upgrade feature,
+    # if set to false all other options are ignored.
     autoUpgrade: true
-    # how many nodes can be upgraded in parallel (default: 1)
-    # 0 means no limit, all nodes will be upgraded in parallel
+    # -- Number of nodes that can be upgraded in parallel (default: 1).
+    # 0 means no limit, all nodes will be upgraded in parallel.
     maxParallelUpgrades: 1
-    # cordon and drain (if enabled) a node before loading the driver on it
+    # -- Cordon and drain (if enabled) a node before loading the driver on it.
     safeLoad: false
-    # options for node drain (`kubectl drain`) before the driver reload
-    # if auto upgrade is enabled but drain.enable is false,
-    # then driver POD will be reloaded immediately without
-    # removing PODs from the node
+    # -- Options for node drain (`kubectl drain`) before the driver reload.
+    # If auto upgrade is enabled but drain.enable is false, then driver POD will be
+    # reloaded immediately without removing PODs from the node.
     drain:
+      # -- Options for node drain (``kubectl drain``) before driver reload, if
+      # auto upgrade is enabled.
       enable: true
+      # -- Use force drain of pods.
       force: true
+      # -- Pod selector to specify which pods will be drained from the node.
+      # An empty selector means all pods.
       podSelector: ""
-      # It's recommended to set a timeout to avoid infinite drain in case non-fatal error keeps happening on retries
+      # -- It's recommended to set a timeout to avoid infinite drain in case
+      # non-fatal error keeps happening on retries.
       timeoutSeconds: 300
+      # -- Delete pods local storage.
       deleteEmptyDir: true
     waitForCompletion:
       # specifies a label selector for the pods to wait for completion
```

Comment:
need to add `--` here ?

Reply:
That's not needed because the field is commented.